setup-machine-arktos.sh

```diff
@@ -78,16 +90,16 @@ echo Setup: Install miscellaneous
 wget -O $HOME/Python-3.8.8.tgz https://www.python.org/ftp/python/3.8.8/Python-3.8.8.tgz
 tar -C $HOME -xzf $HOME/Python-3.8.8.tgz
 cd $HOME/Python-3.8.8
-sudo ./configure
+sudo ./configure --enable-optimizations
 sudo make
-sudo make install
+sudo make altinstall
 sudo ln -sfn /usr/local/bin/python3.8 /usr/bin/python3
 sudo apt remove -fy python3-apt
 sudo apt install -fy python3-apt
 sudo apt update
 sudo apt install -fy python3-pip
-sudo sed -i '1c\#!/usr/bin/python3.8 -Es' /usr/bin/lsb_release
-sudo /usr/local/bin/python3.8 -m pip install --upgrade pip -y -q
+sudo sed -i '1c\#!/usr/bin/python3.6 -Es' /usr/bin/lsb_release
```

Comment:
Nice catch..

Reply:
yes , if we keep it as 3.8 then error occurs during execution of lsb_release command